### DISCUSSION TITLE
Editable table - allow one colum to change type and content of the other

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#11480] [Ideas] [answered]
I am talking about control panel widget, one defined there:
https://github.com/craftcms/cms/blob/develop/src/templates/_includes/forms/editableTable.twig

I was playing around with using it in my plugins, but i wasn't able to achieve one thing - making one column field change the type of another column field.

Lets say that i have columns: "Vehicle type" (dropdown) and second column which is changed depending on vehicle type selected. If car is selected, second column would be "car brand" also with dropdown, listing car brands. If bike is selected, second column would be "bike brand", but this time text field. Second column would also have different `name` attr depending on its content, either `carBrand` or `bikeBrand`.

Is such a thing possible?

**Comment by brandonkelly** [comment, marked as answer]
There’s no way we could achieve this with `editableTable`, as it takes a centralized set of column definitions, and there’s no way we could make it possible to have some rows get one column type, and other rows get a different column type.

**Reply by piotrpog** [reply]
OK, thank you for reply.
